environment.go

```diff
@@ -321,7 +321,6 @@ func (e *envState) createGenerateTasksQueue(ctx context.Context) error {
 
 	remoteQueuGroupOpts := queue.RemoteQueueGroupOptions{
 		Prefix:                    e.settings.Amboy.Name,
-		TTL:                       7 * 24 * time.Hour,
```

Comment:
With the TTL gone, is there a risk that things will be deleted when there is still some work to be done? My goal here was to add some buffer.